admin-rest-api.yaml

```
@@ -919,6 +1405,29 @@ parameters:
     description: The new partition number or the configurations to be updated for the topic.
     schema:
       $ref: '#/definitions/topic_update_request'
+  
+  entity_name:
+    name: entity_name
+    in: path
+    type: string
+    required: true
+    description: The entity name of the quotas can be `default` or `iam-ServiceId`.
+
+  quotas_create:
+    name: quotas_create
+    in: body
+    required: true       
+    description: The new quotas to be created. 
+    schema:
+      $ref: '#/definitions/quotas_create_request'
```

I appreciate that this will require some re-work both to the swagger, and also to the examples based off the generated code - but I would prefer if we used "quota" (singular) over the plural - as the create API only permits a single quota to be created per invocation.
```suggestion
  quota_create:
    name: quota_create
    in: body
    required: true       
    description: The new quota to be created. 
    schema:
      $ref: '#/definitions/quota_create_request'
```